src/Pages/Game/Game.module.css

```diff
@@ -0,0 +1,58 @@
+.container {
+  width: 100%;
+}
+
+.backButton {
+  margin: 1rem 0;
+}
+
+.gameOverview {
+  background: linear-gradient(
+    to left,
+    var(--color-secondary),
+    var(--color-background)
+  );
+  padding: 0 0 1rem;
+}
+
+.image {
+  max-width: 100%;
+}
+
+.gameInfoContainer {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 3;
+}
+
+.gameTitle {
+  font-size: var(--large);
```

Comment:
For better understanding, you could change the variable naming to --font-size-large ...